overlord/snapstate/handlers.go

```diff
@@ -818,6 +852,20 @@ func (m *SnapManager) undoUnlinkCurrentSnap(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
+	// Restore disabled services
+	var missingSvcs []string
+	missingSvcs, err = m.backend.RestoreDisabledServices(oldInfo, snapst.LastActiveDisabledServices, progress.Null)
+	if err != nil {
+		// nothing to undo, since worst case we would have just disabled some
+		// services which will now be removed anyways
```

Comment:
Find this comment a little confusing. The new service definitions were already removed and old ones were just restored. Should undo fail, there's little to be done anyway.

Reply:
Okay, comment updated